scripts/smoke-tests/test_heartbeat.py

```diff
@@ -0,0 +1,34 @@
+import argparse
+
+from playwright.sync_api import sync_playwright
+
+
+def test_heartbeat(base_url: str):
+    """
+    Smoke test to verify that the /heartbeat endpoint is reachable and returns HTTP 200.
+
+    Args:
+        base_url (str): The base URL of the platform.
+    """
+    with sync_playwright() as p:
+        browser = p.chromium.launch()
+        page = browser.new_page()
+
+        response = page.goto(f"{base_url.rstrip('/')}/heartbeat")
```

Comment:
```suggestion
        response = page.goto(f"{base_url.rstrip('/')}/heartbeat?extended=true")
```
you could do this. And also the ingration test could check by different key of dict, other services like celery and forum. So we could have the info if those services are working as expected.

<img width="1439" height="483" alt="image" src="https://github.com/user-attachments/assets/f45b2357-814c-45a0-af9f-96de0130c2ae" />